docs/zynq-7000-getting-started-guide.md

```diff
@@ -0,0 +1,183 @@
+# Setting up and Running a Co-Simulation with a System Clock Example
+
+This demonstration shows how to compile and run the Co-Simulation demo of Buildroot in QEMU with a simulated device in SystemC.  **Buildroot was selected for this demo, but other Linux Kernel's can be built and deployed in QEMU and used as well.** This configuration is tested working for Ubuntu 18.0.4 and assumes that a `cosim` directory is created in your home directory. This walkthrough also assumes that the device being emulated by QEMU is the Xilinx Zynq-7000 SoC.  This SoC seemed like a good candidate but the concept can apply to any QEMU machine which plugs in a compatible remoteport bus interface.
+
+## Dependencies
+
+Below are the dependencies needed to compile all the libraries in this demo:
+
+```bash
+sudo apt update
+sudo apt install cmake gmake gcc qemu-kvm qemu-system qemu-user-static verilator
```

Comment:
The dependencies that we have listed would then need to be installed on your cloud VM. Are you saying that we should update the packages required to match your list or simply giving instructions on how to replicate your process?
I want to make sure I am understanding you correctly.

Reply:
Hi Matt,

I apologize for the very unclear comment!

I tested the guide inside the cloud VM as above to try to mimic the situation where a user that hasn't installed much in his system runs the guide and I ended up needing the apt line I provided above (minus 'gcc' since that is a dependency of 'build-essential', installing 'build-essential' installs gcc).

So looking at the command line in the guide at the moment we shouldn't be needing qemu-kvm qemu-system qemu-user-static (since we are building or own qemu). I was not able to 'apt install gmake' in the VM so I'm guessing it might be an alias to the 'cmake' or 'make' package in your environment(?). Its not super easy to say if some packages on my apt line will be brought in by 'cmake' (recursively through it's dependencies, if it does we can keep 'cmake' on the apt line). The easy way out i think is if you try to build the components with the apt command line I provided (minus gcc) and add in more packages in case you need. For testing that out I provided my test setup above in case it could help out (but if you have another setup for testing it is even better. since then two flows will be tested).

Sorry for the long answer, and thank you again for the hard work :)

Best regards,
Francisco Iglesias